portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp

```diff
@@ -172,19 +173,15 @@
         var converted_oql = oql_html_conversion_vessel.textContent.trim();
         window.QuerySession = window.initDatamanager('<%=geneticProfiles%>'.trim().split(/\s+/),
                                                             converted_oql,
-                                                            ['<%=cancerTypeId%>'.trim()],
-                                                            '<%=samples%>'.trim().split(/\s+/),
+                                                            ['<%=cancerStudyId%>'.trim()],
```

Comment:
Since we have a study-sample sample, why don't we use the study id from that map instead for this?